cli/client/create_lease.go

```diff
@@ -0,0 +1,39 @@
+package client
```

Comment:
I did apply this change. I don't know why it's not showing up an an outdated diff. I think github needs to work some kinks out of this new review / comment system.